src/app/admin/auditLogTable/types.ts

```diff
@@ -1,9 +1,12 @@
 import { Json } from "@/databaseTypesFile";
-import { Schema } from "@/databaseUtils";
+import { ViewSchema } from "@/databaseUtils";
+
+export type AuditLogResponseData = ViewSchema["audit_log_plus"][];
```

Comment:
Shall we put this inside databaseUtils as well?

Reply:
Sure